### ISSUE TITLE
Need more doc

### ISSUE THREAD

**Issue by steph643** [maximummeteor/server-transform#4]
Hi maxnowack, nice work!
Maybe the doc could summarize how the various features are implemented. I heard server-side reactivity usually has poor performances, so having a bit more details about how it works internally would help people make a choice.
For example, I am not sure if adding an extra static field at publish time uses only `observeChanges` or if it is heavier.
Also, it seems your package can be used to achieve reactive joins. Since there already are several packages for this (you might be interested in this document: https://hackpad.com/Meteor-reactive-join-publication-packages-MpSFfsZrMxa), having some kind of comparison would be great.

**Comment by maxnowack** [comment]
Yes, performance is a big problem. After integrating this package in our app and adding some very performance intensive calculations (e.g. aggregating data), the memory and cpu usage raised about 50%.

So we decided to remove this package and find another solution for the calculations.
I've created [maximum:computed-fields](https://atmospherejs.com/maximum/computed-fields) for this case.
Maybe it's also useful for you.

To answer your questions:
This package is using a logic similar to `reywood:publish-composite` and I think it's heavier than only `observeChanges`.
Yes, it's possible to achieve reactive joins. I knowing the document you mentioned, because reactive joins are a big point in our app. We're using `publish-composite` in some cases but mostly a combination of `computed-fields` and template-level subscriptions.

I will update the documentation and add a comparison if I find the time for that.
